SDK-2825: Rework the documentation for evaluator (#1931)
spryker/spryker-docs: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/security.md

## Problem description

> It is possible that project uses the dependencies, that contain known vulnerabilities. To minimise the security risk for the project, such dependencies should be updated till the version, where the vulnerability was fixed.

Reviewer: ```suggestion
A project can sometimes use dependencies that contain known vulnerabilities.. To minimize the security risk for the project, such dependencies should be updated to the version that has the vulnerability fixed.
```

> ## Example of evaluator error message

Reviewer: ```suggestion
## Example of an Evaluator error message
```

```bash
...
{
    "name": "guzzlehttp/psr7",
    "version": "2.4.1",
    "source": {
        "type": "git",
        "url": "https://github.com/guzzle/psr7.git",
        ...
    },
    ...
````

### Resolving the error

To resolve the error:

> 1. Upgrade the package till the version where the issue was fixed.

Reviewer: ```suggestion
1. Upgrade the package to a version where the vulnerability issue is fixed.
```
